### ISSUE TITLE
Ignore dotfiles in commands subdirectories, consider ignoring commands with no description

### ISSUE THREAD

**Issue by fschmittlein** [drud/ddev#1822]
ddev shows files with dot as a custom command

1. ddev

Output:
ddev -h
Create and maintain a local web development environment.                                                                Docs: https://ddev.readthedocs.io
Support: https://ddev.readthedocs.io/en/stable/#support

Usage:
  ddev [command]

Available Commands:
  .gitattributes   .gitattributes (custom web container command)
  .gitattributes   .gitattributes (custom host container command)

**Comment by rfay** [comment]
I think you'll probably find that you created .gitattributes files in the .ddev/commands subdirectories. If you don't need those, delete them and the problem will go away. It's possible that ddev should ignore dotfiles in those directories, not sure.

**Comment by rfay** [comment]
Changed the title.
* It's totally reasonable to ignore dotfiles in commands subdirectories.
* It's probably the right thing to ignore files that don't have "## Usage"
* While we're at it it would make life easier for everybody if command scripts were automatically made executable (chmod +x)

**Comment by gilbertsoft** [comment]
I vote for ignoring dotfiles

**Comment by rfay** [comment]
And adding a .ddev/commands/.gitatributes with
```
* -text eol=lf
```

**Comment by hirasso** [comment]
Hi there! I just noticed that this is happening with ddev version `v1.25.2`, again. Should I create a new issue? When I start ddev:

```
Custom configuration detected in project 'example.com':
  • Web server: .ddev/web-build/Dockerfile
  • Commands:
    - .ddev/commands/.DS_Store
    - .ddev/commands/web/.DS_Store
    - .ddev/commands/web/sync-wordpress-url
  • Docker Compose: .ddev/docker-compose.livereload.yaml
```

**Comment by stasadev** [comment]
Hi @hirasso,

> Should I create a new issue?

No, this will be fixed in the next DDEV release v1.25.3 soon:

- #8418

**Comment by hirasso** [comment]
Nice, thank you for your quick reply! ⚡️
